src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts

```diff
@@ -29,14 +29,8 @@ export class NewsListListViewComponent implements OnInit, AfterViewChecked {
 
   constructor(private renderer: Renderer2) { }
 
-  ngOnInit() {
-    this.textValidationOfMinCharacters();
-  }
+  ngOnInit() {}
```

Comment:
Do you use this lifecycle hook? What is the stile to write empty methods? (and there's **import** and **implements** )

Reply:
Fixed